dso-l2/src/main/resources/logback.xml

```diff
@@ -1,8 +1,8 @@
 <configuration>
 
   <appender name="TC_BASE" class="com.tc.l2.logging.BufferingAppender">
-    <target>System.out</target>
-    <console>true</console>
+    <!--target>System.out</target-->
```

Comment:
We could remove these lines instead of commenting them out. If it makes sense, you could add a line mentioning why this class shouldn't log to console.

Reply:
how about I comment on why they are commented out?  I don't want to remove them just yet.  I'm a little worried some downstream consumer is looking for certain things in the console output for tests.

Reply:
Sure, up to you.